pandas/_libs/tslibs/timestamps.pyx

```diff
@@ -1531,6 +1531,70 @@ class Timestamp(_Timestamp):
             ) from err
         return _dt.ctime()
 
+    def date(self):
+        """
+        Return date object with same year, month and day.
+
+        Examples
+        --------
+        >>> ts = pd.Timestamp('2023-01-01 10:00:00.00')
+        >>> ts
+        Timestamp('2023-01-01 10:00:00')
+        >>> ts.date()
+        datetime.date(2023, 1, 1)
+        """
+        try:
+            _dt = datetime(self.year, self.month, self.day,
+                           self.hour, self.minute, self.second,
+                           self.microsecond, self.tzinfo, fold=self.fold)
```

Comment:
can we construct a `date` object directly? like
```python
_dt = date(self.year, self.month, self.day)
```
, where `date` is imported from `datetime`?

Reply:
might work to do `import datetime as dt`, then use `dt.date`